packages/tempus/src/Tempus_SolutionHistory.cpp

```diff
@@ -18,8 +18,11 @@ namespace Tempus {
 
   // Nonmember constructor from a ParameterList
```

Comment:
This is not the case. Update the doxygen please

Reply:
Done.